snuba/admin/views.py

```diff
@@ -1138,6 +1141,47 @@ def dlq_replay() -> Response:
     return make_response(loaded_instruction.to_bytes().decode("utf-8"), 200)
 
 
+@application.route("/rpc_endpoints", methods=["GET"])
+@check_tool_perms(tools=[AdminTools.RPC_ENDPOINTS])
+def list_rpc_endpoints() -> Response:
+    return Response(
+        json.dumps(RPCEndpoint.list_all_endpoint_names()),
+        200,
+        {"Content-Type": "application/json"},
+    )
+
+
+@application.route("/rpc_execute/<endpoint_name>", methods=["POST"])
+@check_tool_perms(tools=[AdminTools.RPC_ENDPOINTS])
+def execute_rpc_endpoint(endpoint_name: str) -> Response:
+    endpoint_class = RPCEndpoint.get_from_name(endpoint_name, "v1alpha")
```

Comment:
once u address volos comment to not truncate version, you can have users pass the version so it doesnt need to be hard coded